src/test/java/calculator/StringCalculatorTest.java

```diff
@@ -0,0 +1,49 @@
+package calculator;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static calculator.StringCalculator.splitAndSum;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class StringCalculatorTest {
+
+    @Test
+    void 커스텀구분자() {
+        assertThat(splitAndSum("//;\n1;2;3")).isEqualTo(6);
+    }
+
+    @Test
+    void 숫자가_아닌_값() {
+        assertThatThrownBy(() -> splitAndSum("-1,2:a"))
+                .isInstanceOf(RuntimeException.class);
+    }
+
+    @Test
+    void 음수() {
+        assertThatThrownBy(() -> splitAndSum("-1,2:3"))
+                .isInstanceOf(RuntimeException.class);
```

Comment:
같은 Exception 이라도 메세지가 다를 수 있습니다 :) 

```suggestion
        assertThatThrownBy(() -> splitAndSum("-1,2:3"))
                .isInstanceOf(RuntimeException.class)
                .hasMessage("음수는 입력할 수 없습니다.");
```